src/main/java/org/sagebionetworks/web/client/widget/entity/tabs/FilesTabViewImpl.java

```diff
@@ -65,6 +65,8 @@
 	Anchor expandProvenanceLink;
 	@UiField
 	Anchor expandPreviewLink;
+	@UiField
+	Div modalDialogContainer;
```

Comment:
Add the dynamically created Modal to this Div (on preview or provenance graph expand)